lib/Runtime/Library/WabtInterface.cpp

```diff
@@ -0,0 +1,206 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+#include "RuntimeLibraryPch.h"
+
+#ifdef ENABLE_WABT
+#include "../WasmReader/WasmReaderPch.h"
+
+#include "ast-lexer.h"
+#include "common.h"
+#include "ast.h"
+#include "ast-parser.h"
+#include "binary-writer.h"
+#include "binary-writer-spec.h"
+#include "resolve-names.h"
+#include "validator.h"
+#include "Codex/Utf8Helper.h"
+using namespace wabt;
+
+ArenaAllocator* tempArenaPtr;
+struct utf8Allocator
```

Comment:
I totally agree, I didn't like the solution either, but I wanted to make things work and I forgot to clean this up afterward. Thanks for pointing it out